From 60598f3b44237bd5038e33400e749ec1e7e8fbda Mon Sep 17 00:00:00 2001
From: Emekcan Aras <emekcan.aras@arm.com>
Date: Mon, 15 May 2023 10:42:23 +0100
Subject: [PATCH] Platform: corstone1000: Increase BL2 size in flash layout

Increases BL2 size to align with the flash page size in corstone1000.

Signed-off-by: Emekcan Aras <emekcan.aras@arm.com>
Upstream-Status: Submitted [https://review.trustedfirmware.org/c/TF-M/trusted-firmware-m/+/24103]

---
 platform/ext/target/arm/corstone1000/partition/flash_layout.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/platform/ext/target/arm/corstone1000/partition/flash_layout.h b/platform/ext/target/arm/corstone1000/partition/flash_layout.h
index 41b4c6323f..bfe8c4fb3c 100644
--- a/platform/ext/target/arm/corstone1000/partition/flash_layout.h
+++ b/platform/ext/target/arm/corstone1000/partition/flash_layout.h
@@ -89,7 +89,7 @@
 #endif
 
 /* Static Configurations of the Flash */
-#define SE_BL2_PARTITION_SIZE           (0x18800)    /* 98 KB */
+#define SE_BL2_PARTITION_SIZE           (0x19000)    /* 98 KB */
 #define SE_BL2_BANK_0_OFFSET            (0x9000)  /* 72nd LBA */
 #define SE_BL2_BANK_1_OFFSET            (0x1002000)  /* 32784th LBA */
 
